.build/cspell-words.txt

```diff
@@ -1,16 +1,21 @@
 adfs
 Adsi
 amsi
-Antispam
+Andriod
 anob
+Antispam
+Appt
 authsspi
 autodiscover
 bincirc
 BSTR
 cach
 cachtokn
+Callog
+clientinfostring
 Clixml
 cmatch
+Collab
```

Comment:
Fully spell out collaboration. Don't need to fill the dictionary with this type of shorthand.